src/main/java/org/mifos/connector/mojaloop/ilp/IlpConditionHandlerImpl.java

```diff
@@ -45,7 +41,7 @@ public String getILPPacket(String ilpAddress, String amount, Transaction transac
         InterledgerAddress address = InterledgerAddress.builder().value(ilpAddress).build();
         InterledgerPayment.Builder paymentBuilder = InterledgerPayment.builder();
         paymentBuilder.destinationAccount(address);
-        paymentBuilder.destinationAmount(amount);
+        paymentBuilder.destinationAmount(Long.valueOf(amount));
```

Comment:
why Amount is long? it should be BigDecimal or Double

Reply:
its the DTO from interledger-ilp-condition dependency.